### PR TITLE
Add metering integration tests to metering job

### PR DESCRIPTION
**Pull request by chancez**

### CODE REVIEW

**Comment by chancez** [comment]
/hold

I'm verifying this manually still as I updated the `command` to run recently and realized I haven't re-ran tests manually first.

File: ci-operator/jobs/operator-framework/operator-metering/operator-framework-operator-metering-master-presubmits.yaml [hunk 1]
```diff
@@ -33,6 +33,69 @@ presubmits:
...
+          valueFrom:
+            configMapKeyRef:
+              key: operator-framework-operator-metering-master.yaml
+              name: ci-operator-master-configs
+        - name: JOB_NAME_SAFE
+          value: integration-aws-metering
+        - name: TEST_COMMAND
+          value: |
+            set -x
+            # increase the size of the machines the cluster has to be big enough
```
Comment (tschuy) [inline ci-operator/jobs/operator-framework/operator-metering/operator-framework-operator-metering-master-presubmits.yaml]: grammar

**Pull request merged**